src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -386,6 +386,10 @@ public function testParseTagWithEmptyNameThrowsException()
         $loader->load('tag_with_empty_name.xml');
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Since symfony/dependency-injection 5.1: Not setting the attribute "package" of the node "deprecated" is deprecated.
```

Comment:
Labeling a test case `@legacy` means it should be removed in v6. I don't think we want to remove this test (neither the next one), do we? Then we must update it to make it deprecation free. Alternatively, we could have new test case to test the deprecation-free path while keeping the legacy one.